### PR TITLE
Added release-test.yml github workflow script

### PR DESCRIPTION
**Pull request by PrathumP**
- Closes issue #77

### CODE REVIEW

File: .github/workflows/release-test.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  build-and-test:
+    runs-on: [ubuntu-20.04]
+    steps:
+      - uses: actions/checkout@v3
+        
+      - name: Replace file contents
+        run: |
+          # Replace ../src/main.roc with link to latest platform release
+          sed -i 's|../src/main.roc|https://github.com/roc-lang/roc/blob/main/examples/cli/cli-platform/main.roc|g' examples/**/*.roc
```
Comment (Anton-4) [inline .github/workflows/release-test.yml:20]: Thanks for working on this @PrathumP :)
Note that github links like this one will not work with roc for security reasons, I expect that you will need to:
1. use the github api to get the latest pre-release
2. get all assets for that release
3. get every file from the assets that is either a `tar.br` or a `tar.gz` and check which is the most recently created file
4. and then replace the `../src/main.roc` with the link to the file from the previous step

Comment (PrathumP) [reply .github/workflows/release-test.yml]: Got it. I've made a PR but I'm not sure about the changes so please let me know what else needs to be done or corrected.

**Comment by Anton-4** [comment]
The args example failure is a strange one, I'm going to come back to it later.

**Comment by Anton-4** [comment]
TODO: remove pull_request trigger

**Review by Anton-4 — APPROVED** [review]

**Pull request merged**